azure-pipelines/official-build.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
+    branches:
+        include:
+            - v3.x
+    batch: true
+
+# CI only
+pr: none
+
+resources:
+    repositories:
+        - repository: 1esPipelines
+          type: git
+          name: 1ESPipelineTemplates/1ESPipelineTemplates
+          ref: refs/tags/release
+
+extends:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+    parameters:
+        pool:
+            name: 1es-pool-azfunc
+            image: 1es-windows-2022
+            os: windows
+
+        stages:
+            - stage: WindowsUnitTests
+              dependsOn: []
+              jobs:
+                  - template: /azure-pipelines/templates/test.yml@self
+
+            - stage: LinuxUnitTests
+              dependsOn: []
+              jobs:
+                  - template: /azure-pipelines/templates/test.yml@self
+              pool:
+                  name: 1es-pool-azfunc
+                  image: 1es-ubuntu-22.04
+                  os: linux
+
+            - stage: Build
+              dependsOn: []
+              jobs:
+                  - template: /azure-pipelines/templates/build.yml@self
+                    parameters:
+                        IsPreRelease: true
```

Comment:
nit: You can get rid of the IsPreRelease parameter for both official and public builds. That's not used in the durable repo